src/Chapter1.hs

```diff
@@ -478,8 +478,8 @@ Implement a function that returns the last digit of a given number.
   whether it works for you!
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
-
+lastDigit :: Int -> Int
+lastDigit n = mod (abs n) (10 * div (abs n ) 10)
```

Comment:
Creative solution 🎨 

However, it won't work on `-10 < n < 10`, as you will have `mod 5 (10 * div 5 10) = mod 5 (10 * 0) = mod 5 0`  – and this is division on 0 💥 

The solution is a bit simpler 🙂